packages/now-static-build/src/frameworks.ts

```diff
@@ -112,6 +112,27 @@ const frameworkList: Framework[] = [
       },
     ],
   },
+  {
+    name: 'Dojo',
+    slug: 'dojo',
+    dependency: '@dojo/cli',
+    buildCommand: 'dojo build',
+    getOutputDirName: async () => 'output/dist',
```

Comment:
```suggestion
    getOutputDirName: async () => join('output', 'dist'),
```

Does dojo provide a config file to change the output directory? If so then this should read the config file.

Reply:
The output directory is not configurable,  but it does change depending on the type of build.. i.e. if it is a dev build the output get written to `output/dev`. Is there anything that we can do for that?

Reply:
We don't need to account for dev because that will be handled by the `devCommand` in `frameworks.json`.

However, lets change this to `join('output', 'dist')` to make it cross platform.